backend/routes/api/resources.js

```diff
@@ -0,0 +1,248 @@
+const express = require("express");
+const router = express.Router();
+const Resource = require("../../models/Resource");
+const { celebrate, Joi } = require("celebrate");
+const geolib = require("geolib");
+const Fuse = require("fuse.js");
+Joi.objectId = require("joi-objectid")(Joi);
+
+// get all resources
+router.get("/", async (req, res) => {
+  const resources = await Resource.find({});
+  res.json({
+    code: 200,
+    result: resources,
+    success: true
+  });
+});
+
+// get list of resources filtered by location radius
+router.get(
+  "/filter",
+  celebrate({
+    query: {
+      radius: Joi.number(),
+      lat: Joi.number(),
+      long: Joi.number(),
+      keyword: Joi.string()
+    }
+  }),
+  async (req, res) => {
+    const radius = req.query.radius;
```

Comment:
Pro tip for the future: you can use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) to make this clean.